gen/cow/cow.hoon

```diff
@@ -0,0 +1,59 @@
+::  The classic unix "cowsay" implemented in hoon
+::
+::  To see a cow say a quote from gen/cow/quotes.txt run:
+::
+::  +cow/cow
+::
+/-  sole
+/+   generators, pretty-file
+=,  generators
+:-  %ask
+|=  [[now=@da eny=@uvJ bec=beak] ~ ~]
+=/  max-length  80
+=/  file  (pretty-file .^(noun (cat 3 %c %x) `path`/(scot %p p.bec)/[q.bec]/(scot r.bec)/gen/cow/quotes/txt))
+=/  line  (snag (~(rad og eny) (lent file)) file)
+=/  tongue  "U"
+=/  thoughts  "o"
+=/  eyes  "oo"
+=<  (cow)
+|%
```

Comment:
You might want to use `|^` instead, which is like `|%`, except it also takes code that is explicitly executed within the core immediately. So, your `++cow`.

```hoon
|^
  ^-  (sole-request:sole (cask tang))
  etc...
::
++  message
  etc...
--
``